analysis/Analysis/Section_3_4.lean

```diff
@@ -9,20 +9,22 @@ text. When there is a choice between a more idiomatic Lean solution and a more f
 translation, I have generally chosen the latter. In particular, there will be places where the
 Lean code could be "golfed" to be more elegant and idiomatic, but I have consciously avoided
 doing so.
+-/
+namespace Chapter3
+
+export SetTheory (Set Object nat)
+
+variable [SetTheory] (X : Type) (S : _root_.Set X) (f : X → X)
```

Comment:
IMO this is rather confusing because they aren't actually being used by any proofs (which define their own variables), and are thus polluting the context in actual proofs.

<img width="2242" height="540" alt="Screenshot 2025-07-16 at 16 35 01" src="https://github.com/user-attachments/assets/ed967477-b954-4585-92ec-1196bbcdb623" />

If they only exist for the sake of inline comments, is it possible to somehow scope them to the areas with those comments?

Reply:
Ah cool, thanks for the tip! Will tag you if I make one.